win32/build/confutils.js

```diff
@@ -3116,20 +3110,14 @@ function toolset_get_compiler_name(short)
 						When new versions are introduced, adapt also checks in
 						php_win32_image_compatible(), if needed. */
 			name = short ? "VS16" : "Visual C++ 2019";
-		} else if (version >= 1910) {
-			name = short ? "VC15" : "Visual C++ 2017";
-		} else if (version >= 1900) {
-			name = short ? "VC14" : "Visual C++ 2015";
-		} else {
-			ERROR("Unsupported Visual C++ compiler " + version);
 		}
```

Comment:
Should this else block still exist?

Reply:
No, since that is caught already by line 3059ff (`ERROR("Building with MSC_VER " + version + " is no longer supported");`). I have considered to remove the check above, but that would be inconsistent wrt. clang, and checking the clang version a couple of lines below would need to repeat the version string mangling to an integer. And actually, I think it makes more sense to check for an unsupported toolset in `toolset_get_compiler_name()`, but for a supported toolset with an unsupported version in `toolset_get_compiler_version()`.

Reply:
ACK